src/test/java/api/LaunchesApiClient.java

```diff
@@ -0,0 +1,35 @@
+package api;
+
+import com.google.gson.Gson;
+import api.models.launches.ResponseForLaunches;
+import api.models.launches.put.ResponsePutDescription;
+
+import io.restassured.response.Response;
+
+import static api.endpoints.LaunchesApi.LAUNCHES;
+
+public class LaunchesApiClient extends ApiBaseTest {
```

Comment:
Let's create 'clients' or 'api_clients' folder under the 'api' folder and move this class there?

Reply:
Fixed